src/middleware/externalRequests.js

```diff
@@ -294,14 +316,66 @@ const sendMappedObject = (ctx, axiosConfig, request, body) => {
           reqTimestamp,
           responseTimestamp,
           request.id,
-          error
+          error,
+          requestParameters
         )
 
         ctx.orchestrations.push(orchestration)
       }
     })
 }
 
+const addRequestQueryParameters = (ctx, request) => {
+  const requestQueryParams = {}
+
+  if (request.params) {
+    Object.keys(request.params).forEach(param => {
+      const parameterPathStringArray = request.params[`${param}`].path.split(
+        '.'
+      )
+      let parameterValue
+
+      if (parameterPathStringArray[0] === 'payload') {
+        for (let i = 1; i < parameterPathStringArray.length; i++) {
+          const element = parameterPathStringArray[i]
+          if (element) {
+            const splitElement = element.split('')
+
+            if (splitElement.shift() === '[' && splitElement.pop() === ']') {
+              const index = parseInt(splitElement.toString(), 10)
+              parameterValue = parameterValue
+                ? parameterValue[index]
+                : ctx.request.body[index]
+            } else {
+              parameterValue = parameterValue
+                ? parameterValue[`${element}`]
+                : ctx.request.body[`${element}`]
+            }
+          }
+        }
+      } else if (parameterPathStringArray[0] === 'query' && ctx.request.query) {
+        for (let i = 1; i < parameterPathStringArray.length; i++) {
+          parameterValue = parameterValue
+            ? parameterValue[`${parameterPathStringArray[i]}`]
+            : ctx.request.query[`${parameterPathStringArray[i]}`]
+        }
+      }
+
+      if (parameterValue) {
+        const prefix = request.params[`${param}`].prefix
+          ? request.params[`${param}`].prefix
+          : ''
+        const postfix = request.params[`${param}`].prefix
```

Comment:
We seem to be checking for the `prefix` property instead of the `postfix` property.